src/core/server/elasticsearch/scoped_cluster_client.test.ts

```diff
@@ -106,6 +106,27 @@ describe('#callAsCurrentUser', () => {
     );
     scopedAPICaller.mockClear();
 
+    await expect(
+      clusterClient.callAsCurrentUser('security.authenticate', {
+        some: 'some',
+        headers: { additionalHeader: 'Oh Yes!' },
+      })
+    ).resolves.toBe(mockResponse);
+    expect(scopedAPICaller).toHaveBeenCalledTimes(1);
+    expect(scopedAPICaller).toHaveBeenCalledWith(
+      'security.authenticate',
+      { some: 'some', headers: { one: '1', additionalHeader: 'Oh Yes!' } },
+      undefined
+    );
+    scopedAPICaller.mockClear();
+
```

Comment:
I think this should be its own test, not another set of expects within another test.  It will be clearer later on if this fails what it is testing.